idb/postgres/postgres_integration_test.go

```diff
@@ -471,6 +486,9 @@ func TestRekeyBasic(t *testing.T) {
 	pdb, err := idb.IndexerDbByName("postgres", connStr, idb.IndexerDbOptions{}, nil)
 	assert.NoError(t, err)
 
+	err = pdb.LoadGenesis(test.MakeGenesis())
```

Comment:
Are there any tests that don't need the is? Maybe worth wrapping setupPostgres + IndexerDbByName + LoadGenesis in a helper?

Reply:
It's probably a good idea. In another PR?